svo (install the zip, not me).xml

```diff
@@ -15251,9 +15251,11 @@ svo.valid.remove_unknownany(&quot;clumsiness&quot;)</script>
                         <colorTriggerBgColor>#000000</colorTriggerBgColor>
                         <regexCodeList>
                             <string>You lash out clumsily.</string>
+                            <string>^You lash out at (\w+) with (.+), but miss\.$</string>
```

Comment:
A quick grep on some of my old logs turns up plenty of examples of this line applied to denizens, but they generally seem to just be missing due to poor accuracy (low Chivalry skill, to-hit, etc), rather than clumsiness. I don't know if clumsiness also gives this line, or just reduces your accuracy so that you sometimes miss when you otherwise wouldn't. Either way, this line doesn't seem to be a reliable indicator for the affliction.